SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+# SEP-0002 - SunPy Organization Definition
+
+# Abstract
+The SunPy project has changed significantly since the first version of SEP-0002 was written in 2014.
+This SEP details a new governance structure designed to enable the project to grow, while allowing for more input from the wider SunPy community.
+
+## Detailed Description
+The primary role of the SunPy Project is to *facilitate and promote the use and development of community-led, free and open-source solar data-analysis software based on the scientific Python environment*.
```

Comment:
Is this the canoncial location for this definition, or is this definition taken from another SEP? Would be good to have a note if the former, or a link if the latter.

Reply:
I think it's the canonical definition. Feel free to suggest wording to clarify that if you like, I can't think of a non-cludgy way of making that more explicit.